ci-operator/templates/openshift/installer/cluster-launch-installer-kubevirt-e2e.yaml

```diff
@@ -0,0 +1,617 @@
+kind: Template
```

Comment:
/hold

Templates are legacy and we should *not* add new ones. This should be brought in as multi-stage steps (https://steps.ci.openshift.org/help.